devices/Ld2410/Ld2410/Ld2410.cs

```diff
@@ -0,0 +1,379 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO.Ports;
+using System.Threading;
+
+using Ld2410.Commands;
+using Ld2410.Reporting;
+
+namespace Ld2410
+{
+	/// <summary>
+	/// LD2410 radar module client.
+	/// </summary>
+	public sealed class Radar : IDisposable
+	{
+		private readonly SerialPort serialPort;
```

Comment:
All the private properties should start with an underscore `_`
That's what we are adopting